lighthouse-core/config/config-helpers.js

```diff
@@ -122,41 +370,25 @@ function expandAuditShorthand(audits) {
  * @param {string=} configDir
  * @return {Array<LH.Config.AuditDefn>|null}
  */
-function requireAudits(audits, configDir) {
-  const expandedAudits = expandAuditShorthand(audits);
-  if (!expandedAudits) {
+function resolveAuditsToDefns(audits, configDir) {
```

Comment:
I stopped short of converting this one to individual because of the minor annoyance of the `Runner.getAuditList` call, but if everyone would prefer that final step for more complete consistency, I'm game